tools/cellprofiler/starting_modules_nameandtypes.xml

```diff
@@ -0,0 +1,179 @@
+<macros>
+    <xml name="macro_image_type" token_thing="images">
+        <param name="name_to_assign" type="text" value="DNA" label="Name to assign these @THING@" />
+    </xml>
+    
+    <xml name="image_type_condition"> 
+        <conditional name="con_set_intensity">
+            <param name="set_intensity_range_from" type="select" label="Set intensity range from" help="This option determines how the image intensity should be rescaled from 0.0 - 1.0.">
+                <option value="Image metadata">Image metadata</option>
+                <option value="Image bit-depth">Image bit-depth</option>
+                <option value="Manual">Manual</option>
+            </param>
+            <when value="Manual">
+                <param name="maximum_intensity" type="float" value="255.0" label="Maximum intensity"/>
+            </when>
+            <when value="Image metadata" />
+            <when value="Image bit-depth" />
+        </conditional>
+    </xml>
+    <xml name="name_type_rule_matching_file">
+        <param name="operator" type="select">
+            <option value="does">Does</option>
+            <option value="doesnot">Does not</option>
+        </param>
+        <param name="contain" type="select">
+            <option value="contain">Contain</option>
+            <option value="Contain regular expression">Contain regular expression</option>
+            <option value="startwith">Start with</option>
+            <option value="endwith">End with</option>
+            <option value="Exactly match">Exactly match</option>
+        </param>
+        <param name="match_value" type="text"/>
+    </xml>
+    <xml name="name_type_rule_image">
+        <param name="operator" type="select">
+            <option value="does">Is</option>
+            <option value="doesnot">Is not</option>
+        </param>
+        <param name="match_type" type="select">
+            <option value="iscolor">Color</option>
+            <option value="ismonochrome">Monochrome</option>
+            <option value="isstack">Stack</option>
+            <option value="isstackframe">Stack frame</option>
+        </param>
+    </xml>
+    <xml name="name_type_rule_metadata">
+        <param name="operator" type="select">
+            <option value="does">Does</option>
+            <option value="doesnot">Does not</option>
+        </param>
+        <param name="match_type" type="select">
+            <option value="FileLocation">Have FileLocation matching</option>
+            <option value="Frame">Have Frame matching</option>
+            <option value="Screen">Have Screen matching</option>
+            <option value="Series">Have Series matching</option>
+        </param>
+        <param name="match_value" type="text"/>
+    </xml>
+    <xml name="name_type_rule_ext">
+        <param name="operator" type="select">
+            <option value="does">Is</option>
+            <option value="doesnot">Is not</option>

```

Comment:
are the values in the ccpipe "does" and "doesnot"?

Reply:
yes.. value needs to be does and doesnot